.gitmodules

```diff
@@ -6,4 +6,4 @@
 	url = ../../fluent/fluent-bit.git
 [submodule "submodules/opentelemetry-operations-collector"]
 	path = submodules/opentelemetry-operations-collector
-	url = ../../GoogleCloudPlatform/opentelemetry-operations-collector.git
+	url = git@github.com:observIQ/opentelemetry-operations-collector.git
```

Comment:
This is temporary, and will be reverted when the underlying PR is merged.

Reply:
You can use an `insteadOf` in your Git config to override this without changing the `.gitmodules` file.

Reply:
underlying PR is merged :)

Reply:
This change has been reverted, and the submodule has been updated accordingly.